src/main/java/edu/ucar/unidata/rosetta/service/FileParserManagerImpl.java

```diff
@@ -21,7 +23,7 @@
 
     protected static Logger logger = Logger.getLogger(FileParserManagerImpl.class);
 
-    public List<List<String>> parsedFileData = new ArrayList<List<String>>();
+    public List<List<String>> parsedFileData = new ArrayList<>();
     public List<String> header = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make header a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)